src/beamlime/kafka/message_adapter.py

```diff
@@ -343,6 +343,3 @@ def get_messages(self) -> Sequence[U]:
                 if self._raise_on_error:
                     raise
         return adapted
-
-    def close(self) -> None:
-        self._source.close()
```

Comment:
The `close` mechanism was removed, since it was redundant with the `close()` calls from the contextmanagers used to create the Kafka consumers.